cmd/s3-peer-router.go

```diff
@@ -27,7 +27,7 @@ const (
 )
 
 type s3PeerAPIHandlers struct {
-	*localBMS
+	bms *localBMS
```

Comment:
the right way is to use the interface itself as part of the struct rather than the `*localBMS`